sw/host/opentitanlib/src/io/uart.rs

```diff
@@ -9,6 +9,7 @@ use std::time::Duration;
 use anyhow::Result;
 use clap::Args;
 use serde::{Deserialize, Serialize};
+use serialport::Parity;
```

Comment:
Could you make this a `pub use`, so that `opentitantool` or other users of `io::uart` do not need to directly refer to the `serialport` library, when using our io traits.

Reply:
That's a good idea, will do